src/learnerPortalLinks.js

```diff
@@ -29,10 +29,18 @@ async function fetchLearnerPortalLinks(apiClient, userId) {
       const enterpriseCustomer = enterpriseCustomers[i];
       const enterpriseCustomerSlug = enterpriseCustomer.slug;
       const enableLearnerPortal = enterpriseCustomer.enable_learner_portal;
+      const brandingConfiguration = enterpriseCustomer.branding_configuration;
       if (enableLearnerPortal && enterpriseCustomerSlug) {
         learnerPortalLinks.push({
           title: `${enterpriseCustomer.name}`,
           url: `${window.location.protocol}//${learnerPortalHostname}/${enterpriseCustomerSlug}`,
+          // branding_configuration is not always returned as part
+          // of the response so check if it exists before referencing fields
+          branding_configuration: brandingConfiguration ? {
+            logo: brandingConfiguration.logo,
+            banner_border_color: brandingConfiguration.banner_border_color,
+            banner_background_color: brandingConfiguration.banner_background_color,
+          } : null,
```

Comment:
When I did work on learner portal enterprise I think I also had a case where brandingConfiguration was non null but some of the fields were, so I had to default them.. I do not know if that is a requirement here, is it?

for ref: https://github.com/edx/frontend-app-learner-portal-enterprise/blob/35ae057a7999a23f07d8a70c7a7acc1e9dfc3409/src/components/enterprise-page/data/hooks.js#L35

Reply:
Great catch, realizing that all three of these fields are actually nullable so in order to keep the keys present I'll add a fallback.